languages/Dockerfile.compile.j2

```diff
@@ -1,12 +1,13 @@
 FROM {{builder_image}} as builder
-USER algo
-COPY --chown=algo algosource /opt/algorithm/
+USER 2222
+COPY --chown=2222 algosource /opt/algorithm/
+ENV HOME=/home/algo
```

Comment:
little curious why env var needed to be set

Reply:
pip install --local was otherwise trying to write to /.local and it doesnt have permissions to do so.

Reply:
Kaniko sets the value of HOME to / if the user is a UID and not a username - https://github.com/algorithmiaio/kaniko/blob/master/pkg/commands/run.go#L134-L141.  I'm actually still poking around with this because there's some weird file permissions on /home/algo in some cases (depending on whether caching was used and whether the image was in the cache or not) :(